variables.tf

```
@@ -4,30 +4,55 @@
 # You must provide a value for each of these parameters.
 # ------------------------------------------------------------------------------
 
-variable "subnet_id" {
-  type        = string
-  description = "The ID of the AWS subnet to deploy into (e.g. subnet-0123456789abcdef0)."
+variable "users" {
+  description = "A map whose keys are the usernames of each non-admin user and whose values are a map containing supported user attributes.  The only currently-supported attribute is \"require_mfa\" (boolean).  Example: { \"firstname1.lastname1\" = { \"require_mfa\" = false }, \"firstname2.lastname2\" = { \"require_mfa\" = true }, \"firstname3.lastname3\" = { \"require_mfa\" = false } }"
+  type        = map(map(string))
```

This variable is currently of type `map(map(bool))` and would become more complicated if [this suggestion](xxx) were implemented.  I think it makes more sense to define it more precisely:
```suggestion
variable "users" {
  description = "A map whose keys are the usernames of each non-admin user and whose values are an object containing supported user attributes.  The only currently-supported attribute is \"require_mfa\" (boolean).  Example: { \"firstname1.lastname1\" = { \"require_mfa\" = false }, \"firstname2.lastname2\" = { \"require_mfa\" = true }, \"firstname3.lastname3\" = { \"require_mfa\" = false } }"
  type        = map(object({require_mfa = bool}))
```

---

Please see [5c7c918](https://github.com/cisagov/cyhy-users-non-admin/pull/5/commits/5c7c918fd83aa08c8ca748cecbb16d997d0eecc2).